### PR TITLE
Replaced block quotes with emphasis; bold warnings, italic info

### LINKED ISSUE #509: Show warning in bold texts in the README
A user suggested we display warnings in bold text, because block quotes are rendered in gray color, which does not catch the eye enough.

--- 

Unbelievable i overlooked it... perhaps because it is gray (comment).

> ⚠ You must rebuild the stack images with docker-compose build whenever you switch branch or update the version of an already existing stack.

Perhaps better write your ⚠ in **bold** instead of a comment 😅 something like:

> **⚠ You must rebuild the stack images with `docker-compose build` whenever you switch branch or update the version of an already existing stack.**

_Originally posted by @FireEmerald in https://github.com/deviantony/docker-elk/issues/508#issuecomment-641804865_

### PR DESCRIPTION
**Pull request by bubbajames-docker**
Replaced block quotes with emphasis; bold for warnings, italics for information sources.

509-warnings-bold
509-info-italic

Closes #509

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,9 +9,7 @@ Run the latest version of the [Elastic stack][elk-stack] with Docker and Docker
 It gives you the ability to analyze any data set by using the searching/aggregation capabilities of Elasticsearch and
 the visualization power of Kibana.
 
-> :information_source: The Docker images backing this stack include [Stack Features][stack-features] (formerly X-Pack)
```
Comment (antoineco) [inline README.md:12]: Can we please stick to the current formatting for consistency? (line width = 120)

File: README.md [hunk 2]
```diff
@@ -25,35 +23,39 @@ Other available stack variants:
 
 ## Contents
 
-1. [Requirements](#requirements)
```
Comment (antoineco) [inline README.md:28]: Redesigning the entire Table of Contents is out of this PR's scope.

**Review by antoineco — CHANGES_REQUESTED** [review]
Looking good, thanks for tackling this! Please consider reverting the ToC and formatting changes, and then we're good for a merge :+1:

**Comment by bubbajames-docker** [comment]
Review comments have been addressed.

1. Reverted TOC  
2. Restored 120 column limits.

File: README.md (hunk 2 above)
Comment (bubbajames-docker) [reply README.md]: Sorry about that, it was not an intended change.  VS Code markdown editor auto-updated TOC.  Fixed.

File: README.md (hunk 1 above)
Comment (bubbajames-docker) [reply README.md:12]: Fixed.

**Review by antoineco — APPROVED** [review]

**Comment by antoineco** [comment]
Thanks again @bubbajames-docker 🙌

**Pull request merged**